recordings-downloader/config.yml

```diff
@@ -12,7 +12,7 @@ steps:
   content: |
     For this tutorial, the user will be asked to input the credentials through a simple console input.
 
-    The user will be also asked for the date interval of the recordings. Please take note of the format for the date input (YYYY-MM-DDThh:mm:ss/YYYY-MM-DDThh:mm:ss)
+    The user will be also asked for the date interval of the recordings. Please take note of the format for the date input `(YYYY-MM-DDThh:mm:ss/YYYY-MM-DDThh:mm:ss)`.
```

Comment:
Since this is now being expressed as a literal value/format, remove the parens (unless the actual value requires them, of course).